.github/workflows/pr-actions.yml

```diff
@@ -118,3 +118,61 @@ jobs:
           gh pr comment $PR_NUM -b "fix:${PR_ACTION} failed or was cancelled. For details, see $GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID."
         env:
           GH_TOKEN: ${{ secrets.OPENTELEMETRYBOT_GITHUB_TOKEN }}
+
+  nudge-action:
+    name: Rerun PR checks
+    runs-on: ubuntu-latest
+
+    if: |
+      github.event.issue.pull_request &&
+      github.event.comment.body == '/nudge'
+    permissions:
+      contents: read
+      pull-requests: write
+
+    steps:
+      - name: Context info
+        run: |
+          echo $PR_NUM
+          echo $COMMENT
+        env:
+          PR_NUM: ${{ github.event.issue.number }}
+          COMMENT: ${{ github.event.comment.body }}
+
+      - name: Write start comment
+        run: |
+          gh pr comment $PR_NUM -b "You triggered /nudge action run at $GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID"
+        env:
+          GH_TOKEN: ${{ secrets.OPENTELEMETRYBOT_GITHUB_TOKEN }}
+          PR_NUM: ${{ github.event.issue.number }}
+
+      - name: Rerun PR checks
+        run: |
+          # Get the latest check run to re-run all checks
+          gh pr checks $PR_NUM --repo $GITHUB_REPOSITORY --watch
+          # This doesn't actually re-run them, but shows the status
+          # Now re-run them
+          gh run rerun $(gh run list --branch $(gh pr view $PR_NUM --json headRefName -q .headRefName) --limit 1 --json databaseId -q '.[0].databaseId')
```

Comment:
There's a lot going on in this line. Could you factor out the fetching of argument values, possibly putting them in an `env:` variable for this step?

E.g., I did this locally or the (current) stuck PR #6574:

```console
$ PR_NUM=6574 
$ BRANCH=$(gh pr view $PR_NUM --json headRefName -q .headRefName)
$ RUNID=$(gh run list --branch $(gh pr view $PR_NUM --json headRefName -q .headRefName) --limit 1 --json databaseId -q '.[0].databaseId')
$ gh run rerun $RUNID                                                                                                                    
✓ Requested rerun of run 13966456806
```

But nothing has been happening to the PR on GH.

Reply:
Btw, I've rebased #6574 (because I wanted to see a clean check status), so it isn't stuck anymore.